scripts/js/lib/api/releaseNotes.ts

```diff
@@ -172,6 +174,19 @@ New features, bug fixes, and other changes in previous versions of ${pkg.title}.
   return markdown;
 }
 
+export function generateReleaseNotesIndex(pkg: Pkg): string {
+  const grouped = groupByMajorVersion(
+    pkg.releaseNotesConfig.separatePagesVersions,
+  );
+  const htmlSections = Object.entries(grouped)
+    .map(([majorVersion, versionList]) =>
+      renderVersionGroup(majorVersion, versionList),
+    )
+    .join("\n\n");
+
+  return htmlSections;
```

Comment:
Nice job with this! Good functional programming style

```suggestion
  return Object.entries(grouped)
    .map(([majorVersion, versionList]) =>
      renderVersionGroup(majorVersion, versionList),
    )
    .join("\n\n");

```